### ISSUE TITLE
Re-implement existing DROID functionality in clojure

### ISSUE THREAD

**Issue by lmcmicu** [ontodev/droid#16]

**Comment by jamesaoverton** [comment]
Some context for this: I wrote the first prototype for DROID in Clojure, which is a language that I love but don't get to use very much. People are afraid of parentheses, so when our focus is on collaboration we use Python and Java.

The prototype started growing toward ontology-project-specific customizability. So in order to address a wide audience of customizers, I rewrote the prototype in Python.

Then, in discussion with @cmungall and others, I decided to strictly narrow the scope of DROID to a build system (GNU Make) plus a version control system (GitHub), and push the customizations out. (Now I want to try implementing custom functionality with CGI #15, and created a new repo for ontology scripts that I'll reuse: https://github.com/ontodev/gizmos.)

DROID now has a very narrow scope and a short list of planned features. While it would be nice to get a bunch of users, I don't expect a wide community of contributors. So my main reason for implementing DROID in Python is gone. For this use case Clojure has easier deployment (one JAR) and better performance (JVM) than Python, but ultimately I just want to use Clojure for this project. So that's the new plan.
